client/src/js/directives/bhFindSupplier.js

```diff
@@ -0,0 +1,201 @@
+angular.module('bhima.directives')
```

Comment:
I think we actually have a module `bhima.components`.  This belongs in the `bhima.components` module, not the directives module.